exercises/hello-world/src/example.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
```

Comment:
I need to add that include to the test, though, since it uses `NULL` without explicitly including `stddef.h`.